airbrakes/mock/display.py

```diff
@@ -128,23 +130,23 @@ def update_display(self) -> None:
         Updates the display with real-time data. Runs in another thread. Automatically stops when
         the simulation ends.
         """
+        # Don't print the flight data if we are in debug mode
+        if self._args.debug:
+            return
+
         while self._running:
-            if self.end_sim_natural.is_set():
-                self._update_display(self.NATURAL_END)
-                break
-            if self.end_sim_interrupted.is_set():
-                self._update_display(self.INTERRUPTED_END)
-                break
-            if not self._args.mock and self._airbrakes.state.name == "MotorBurnState":
-                self._update_display(self.STATE_END)
-                break
-            # Don't print the flight data if we are in debug mode
-            if not self._args.debug:
-                self._update_display(False)
-            else:
-                break
-
-    def _update_display(self, end_sim: Literal["natural", "interrupted"] | bool = False) -> None:
+            self._update_display(False)
+
+        if self.end_sim_natural.is_set():
+            self._update_display(self.NATURAL_END)
+        if self.end_sim_interrupted.is_set():
+            self._update_display(self.INTERRUPTED_END)
+        if not self._args.mock and self._airbrakes.state.name == "MotorBurnState":
+            self._update_display(self.STATE_END)
+
+    def _update_display(
```

Comment:
if we are passing a string literal here, it seems weird to also have the option to pass in a boolean, especially if we are never really setting this boolean to true.

Also idk how much I like string literals. Would maybe prefer enums.

Reply:
Yeah I'm changing this to use constants like `1`, `2`, etc.

.
.
.

Hahaha jk. Enums it is.